man/man8/zfs.8

```diff
@@ -487,6 +502,17 @@ been set somewhere in the tree under which the dataset resides.
 
 .sp
 .ne 2
+.mk
+.na
+\fB\fBkeystatus\fR
+.ad
+.sp .6
+.RS 4n
+Indicates if an encryption key is currently loaded into ZFS. The possible values are \fBnone\fR, \fBavailable\fR, and \fBunavaliable\fR. See \fBzfs key -l\fR and \fBzfs key -u\fR.
```

Comment:
The ZOL manpage and IllumOS look quite different, so I had to insert the next by hand, and best-guess the formatting around it. I can handle the small manpage fixes, and white space issues.